<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowStrategicAllForm</class>
 <widget class="GUIFormBase" name="ShowStrategicAllForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>76</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Show Strategic all</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>CurrentStrategicNumber</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCurrentStrategicNumber">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonShow">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
   <property name="icon">
    <iconset resource="ShowStrategicAllNumber.qrc">
     <normaloff>:/Qtres/Preview.png</normaloff>:/Qtres/Preview.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>CurrentStrategicNumberForSeq</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCurrentStrategicNumberForSeq">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>CurrentStrategicNumberForCalc</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCurrentStrategicNumberForCalc">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ShowStrategicAllNumber.qrc"/>
 </resources>
 <connections/>
</ui>
